frontend/src/app/shared/services/countdown.service.ts

```diff
@@ -18,15 +18,18 @@ export class CountdownService {
           new Date(now.getFullYear(), now.getMonth(), now.getDate(), hour, minute, 0),
           ts => {
             // if minutes or seconds are < 10 add a 0 before
```

Comment:
This comment needs an update as we'll have the hours too

`// if minutes, seconds or hours are < 10 add a 0 before`